oci/table_oci_cloud_guard_detector_recipe.go

```diff
@@ -0,0 +1,284 @@
+package oci
+
+import (
+	"context"
+	"strings"
+
+	"github.com/oracle/oci-go-sdk/v36/cloudguard"
+	oci_common "github.com/oracle/oci-go-sdk/v36/common"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableCloudGuardDetectorRecipe(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "oci_cloud_guard_detector_recipe",
+		Description: "OCI Cloud Guard Detector Recipe",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getCloudGuardDetectorRecipe,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCloudGuardDetectorRecipes,
+		},
+		GetMatrixItem: BuildCompartementRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "DisplayName of detector recipe.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+			{
+				Name:        "id",
+				Description: "Ocid for detector recipe.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "source_detector_recipe_id",
+				Description: "Recipe Ocid of the Source Recipe to be cloned.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The current state of the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+
+			// other columns
+			{
+				Name:        "description",
+				Description: "Detector recipe description.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "The date and time the detector recipe was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated").Transform(convertDateToTime),
+			},
+			{
+				Name:        "time_updated",
+				Description: "The date and time the detector recipe was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeUpdated").Transform(convertDateToTime),
```

Comment:
```suggestion
				Transform:   transform.FromField("TimeUpdated.Time"),
```
can we do like this ?